README.md

```diff
@@ -675,3 +675,42 @@ You don't need to build anything to be able to run locally, but if you need the
 ```text
 poetry build
 ```
+
+### Development - Version release
+
+You can check the current version on master with these commands:
+```text
```

Comment:
you can remove the `text` language directive

Reply:
done